qiskit/circuit/library/standard_gates/h.py

```diff
@@ -107,7 +107,15 @@ def control(
         return gate
 
     def inverse(self, annotated: bool = False):
-        r"""Return inverted H gate (itself)."""
+        r"""Return inverted H gate (itself).
+
+        Args:
+            annotated: indicates whether the inverse gate can be implemented
+                as an annotated gate.
```

Comment:
```suggestion
            annotated: Typically when set to ``True`` this is used to return an :class:`.AnnotatedOperation` 
                with an inverse modifier set instead of a concrete :class:`.Gate`. However, for this class this
                argument is ignored as this gate is self inverse.
```